src/components/atoms/ChatMessage/ChatMessage.scss

```diff
@@ -115,10 +117,16 @@ $reply-icon-radius: 20px;
 
     border-radius: $border-radius--max;
     background-color: $secondary--light;
+    transition: opacity 100ms;
+    border: none;
```

Comment:
Why transitioning `opacity`? Why not using the standard transition function variable? Why border none?